src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -1,27 +1,167 @@
+        Persistence.apiUsers.removeById(apiUser.id);
+    }
+
+    /**
+     * Ensure that an {@link ApiUser} can create an API key for self.
+     */
+    @Test
+    public void canCreateApiKeyForSelf() {
+        assumeTrue(getBooleanEnvVar("RUN_E2E"));
+        HttpResponse<String> response = createApiKeyRequest(apiUser.id, apiUser.auth0UserId);
+        Assertions.assertEquals(200, response.statusCode());
+        ApiUser userFromResponse = JsonUtils.getPOJOFromJSON(response.body(), ApiUser.class);
+        // refresh API key
+        ApiUser userFromDb = Persistence.apiUsers.getById(apiUser.id);
+        LOG.info("API user successfully created API key id {}", userFromResponse.apiKeys.get(0).id);
+        Assertions.assertEquals(userFromDb.apiKeys, userFromResponse.apiKeys);
+    }
+
+    /**
+     * Ensure that an {@link AdminUser} can create an API key for an {@link ApiUser}.
+     */
+    @Test
+    public void adminCanCreateApiKeyForApiUser() {
+        assumeTrue(getBooleanEnvVar("RUN_E2E"));
+        HttpResponse<String> response = createApiKeyRequest(apiUser.id, adminUser.auth0UserId);
+        Assertions.assertEquals(200, response.statusCode());
+        ApiUser userFromResponse = JsonUtils.getPOJOFromJSON(response.body(), ApiUser.class);
+        // refresh API key
+        ApiUser userFromDb = Persistence.apiUsers.getById(apiUser.id);
+        LOG.info("Admin user successfully created API key id {}", userFromResponse.apiKeys.get(0).id);
+        Assertions.assertEquals(userFromDb.apiKeys, userFromResponse.apiKeys);
+    }
+
+    /**
+     * Ensure that an {@link ApiUser} can delete an API key for self.
+     */
+    @Test
+    public void canDeleteApiKeyForSelf() {
+        assumeTrue(getBooleanEnvVar("RUN_E2E"));
+        ensureAtLeastOneApiKeyIsAvailable();
+        // delete key
+        String keyId = apiUser.apiKeys.get(0).id;
+        HttpResponse<String> response = deleteApiKeyRequest(apiUser.id, keyId, apiUser.auth0UserId);
+        Assertions.assertEquals(200, response.statusCode());
+        ApiUser userFromResponse = JsonUtils.getPOJOFromJSON(response.body(), ApiUser.class);
+        Assertions.assertTrue(userFromResponse.apiKeys.isEmpty());
+        LOG.info("API user successfully deleted API key id {}", keyId);
+        // refresh API key
+        ApiUser userFromDb = Persistence.apiUsers.getById(apiUser.id);
+        Assertions.assertTrue(userFromDb.apiKeys.isEmpty());
+    }
+
+    /**
+     * Ensure that an {@link AdminUser} can delete an API key for an {@link ApiUser}.
+     */
+    @Test
+    public void adminCanDeleteApiKeyForApiUser() {
+        assumeTrue(getBooleanEnvVar("RUN_E2E"));
+        ensureAtLeastOneApiKeyIsAvailable();
+        // delete key
+        String keyId = apiUser.apiKeys.get(0).id;
+        HttpResponse<String> response = deleteApiKeyRequest(apiUser.id, keyId, adminUser.auth0UserId);
+        Assertions.assertEquals(response.statusCode(), 200);
```

Comment:
It looks like you swapped the arguments. 200 should be the first arg.

Reply:
I would just write `assertEquals(...)` and import `Assertions` with an `import static org.junit.jupiter.api.Assertions.assertEquals;`